multi-arch/samples.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Samples-Non-X86
+authors:
+  - "@gabemontero"
+reviewers:
+  - "@smarterclayton"
+  - "@crawford"
+  - "@bparees"
+  - "@adambkaplan"
+approvers:
+  - "@smarterclayton"
+  - "@crawford"
+  - "@bparees"
+creation-date: 2019-09-20
+last-updated: 2019-09-20
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Sample Imagestream and Templates on non-x86 Platforms
+
+As of this writing, all samples are provided and owned by non-OpenShift 
+Development teams within Red Hat.  And none of those teams 
+
+ - provide host images on registry.redhat.io or quay.io that support PPC or Z 
```

Comment:
i think a some are being built for PPC, e.g.:
https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/rhoar-nodejs/nodejs-8

Available CPU Architectures | AMD64PPC64LE

but certainly not Z.

Reply:
Yeah but they are not included in openshift/library ... I'll clarify